StorageManager.js

```diff
@@ -0,0 +1,29 @@
+class StorageManager {
+  constructor(){
+    this.usersStorage = JSON.parse(localStorage.getItem('users')) || []
+  }
+  addToStorage = (userData) => {
+    const user = userData.user
+    if(user){
+      const username = `${user.firstName} ${user.lastName}`
+      for(let storedUser of this.usersStorage)
+        if(storedUser.username === username) return     
+      this.usersStorage.push({username,userData})
```

Comment:
inside userData there is. the user's name - so why do you need the username property inside this object? you could make the userData object to having the user name in a more obvious or comfortable location